doc/conf.py

```diff
@@ -110,10 +110,11 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-#html_title = None
+# Don't show version in title
+html_title = "FOSHttpCache Documentation"
```

Comment:
This hides the version from the title when viewing the docs. Currently foshttpcache.readthedocs.org is listed as ‘FOSHttpCache 1.0.0 documentation’ which may be somewhat confusing as we’re already at 1.3.0.